it/uk/gov/hmrc/partnershipidentificationfrontend/utils/ComponentSpecHelper.scala

```diff
@@ -134,6 +134,6 @@ trait ComponentSpecHelper extends AnyWordSpec with Matchers
                           signOutUrl: String,
```

Comment:
Would it not be better to pass "businessVerificationCheck" in as a method argument?

Reply:
why??? ... nobody is using with value of false